ocaml_version.ml

```diff
@@ -198,6 +198,8 @@ module Releases = struct
 
   let recent_with_dev = List.concat [recent; dev]
 
+  let uses_options_packages t =
+    t.major > 4 || t.major = 4 && t.minor >= 12
```

Comment:
```suggestion
    (t.major, t.minor) >= (4, 12)
```

Reply:
I obviously have my 640KB base RAM in mind... your version allocates a block and calls polymorphic compare 😃